hibernate-core/src/test/java/org/hibernate/orm/test/query/criteria/CriteriaMultiselectGroupByAndOrderByTest.java

```diff
@@ -132,13 +133,15 @@ public void testSubqueryGroupBy(SessionFactoryScope scope) {
 	@Test
 	@Jira( "https://hibernate.atlassian.net/browse/HHH-17231" )
 	@SkipForDialect( dialectClass = SybaseASEDialect.class, reason = "Sybase doesn't support order by + offset in subqueries")
+	@SkipForDialect( dialectClass = AltibaseDialect.class, reason = "Altibase offset in limit clause starts from 1")
```

Comment:
Is this still correct? It seems you adapted the `SqlAstTranslator` to respect this difference.

Reply:
Overriding prepareLimitOffsetParameters() in SqlAstTranlator does not affect offset when offset is used as belows. Could you please let me know the ways to modify offset value?

subquery.orderBy(
   cb.desc( entityName )
).offset( 0 );

Reply:
See the comment I left in your translator.